scripts/ec2/describe_addresses.coffee

```diff
@@ -0,0 +1,26 @@
+# Description:
+#   List elastic IP's
+#
+# Commands:
+#   hubot ec2 describe addresses
+
+aws = require('../../aws').aws()
+ec2 = new aws.EC2()
+
+module.exports = (robot) ->
+  robot.respond /ec2 describe addresses$/i, (msg) ->
+    msg.send "Fetching ..."
+    ec2.describeAddresses {}, (err,response) ->
+      if err
+        msg.send err
+      else
+        keys = Object.keys(response.Addresses[0])
```

Comment:
The output keys depends on only first Addresses[0].
This causes the detail information about Addresses[1:] is hidden.

[GOOD] 52.197.83.53 is an associated address, and the information is found.

```
hubot> hubot ec2 describe addresses
Fetching ...
InstanceId | PublicIp | AllocationId | AssociationId | Domain | NetworkInterfaceId | NetworkInterfaceOwnerId | PrivateIpAddress
i-515777ce | 52.197.83.53 | eipalloc-320a4a57 | eipassoc-41896126 | vpc | eni-e2a276a8 | 554519655767 | 10.0.1.160 |
UNASSOCIATED  | 52.69.131.105 | eipalloc-210f4f44 | UNASSOCIATED  | vpc | UNASSOCIATED  | UNASSOCIATED  | UNASSOCIATED  |
```

[NOT GOOD] 52.197.122.115 is an associated address, but the information is not found.

```
hubot> hubot ec2 describe addresses
Fetching ...
PublicIp | AllocationId | Domain
52.197.83.53 | eipalloc-320a4a57 | vpc |
52.197.122.115 | eipalloc-860848e3 | vpc |
```